### PR TITLE
[IMP] sale: ux improvements on preview quotations

### PR DESCRIPTION
**Pull request by mitali-willdooit**
Description of the issue/feature this PR addresses:
<b> Task:</b> https://www.odoo.com/web?#id=2002587&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720
<b> Pad:</b> https://pad.odoo.com/p/r.604d985c5e61bc6fcb470e1e0b92b94c
Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale_management/views/sale_portal_templates.xml [hunk 1]
```diff
@@ -13,16 +13,9 @@
             <t t-set="display_remove" t-value="any([line.sale_order_option_ids for line in sale_order.order_line])"/>
         </xpath>
 
-        <xpath expr="//section[@id='details']//table[@id='sales_order_table']/thead/tr" position="inside">
```
Comment (dbo-odoo) [inline addons/sale_management/views/sale_portal_templates.xml]: I think this part needs to be merged in the basic template, no completely removed (cfr comment on task)

File: addons/sale_management/views/sale_portal_templates.xml [hunk 2]
```diff
@@ -14,15 +14,14 @@
...
             <th t-if="display_remove">
-                <!-- TODO: remove in master -->
-                <t t-set="colspan" t-value="colspan+1"/>
             </th>
         </xpath>
 
         <xpath expr="//section[@id='details']//t[@t-if='not line.display_type']" position="inside">
             <td class="text-center" t-if="display_remove">
-                <a t-att-href="sale_order.get_portal_url(suffix='/update_line/%s' % line.id, query_string='&amp;unlink=True')" class="mb8 js_update_line_json d-print-none" t-if="line.sale_order_option_ids" aria-label="Remove" title="Remove">
+                <a t-att-href="sale_order.get_portal_url(suffix='/update_line/%s' % line.id, query_string='&amp;unlink=True')" class="mb8 js_update_line_json d-print-none" t-if="sale_order and sale_order.state in ('draft', 'sent') and line.sale_order_option_ids" aria-label="Remove" title="Remove">
```
Comment (dbo-odoo) [inline addons/sale_management/views/sale_portal_templates.xml]: unnecessary condition: when would `sale_order` be falsy?

Comment (dbo-odoo) [reply addons/sale_management/views/sale_portal_templates.xml]: and the `display_remove` variable in the qweb template at line 13 should use the same condition (it should be false if the SO is not in state `draft` or `sent`)

File: addons/sale_management/views/sale_portal_templates.xml (hunk 1 above)
Comment (mpa-odoo) [reply addons/sale_management/views/sale_portal_templates.xml]: @dbo Done

File: addons/sale_management/views/sale_portal_templates.xml (hunk 2 above)
Comment (mpa-odoo) [reply addons/sale_management/views/sale_portal_templates.xml]: @dbo-odoo  Done

**Comment by dbo-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by dbo-odoo** [comment]
@robodoo retry

**Comment by dbo-odoo** [comment]
On Jul 8, 2019 9:37 PM, Robot Odoo <notifications@github.com> wrote:Staging failed: ci/runbot on 6272919facc13f10ba884770b3a79ddd168492e8 (view more at http://runbot.odoo.com/runbot/build/558578)

**Comment by rdeodoo** [comment]
@dbo-odoo You might want to retry this one, but as you gave the finger to robodoo I think you are screwed

**Comment by dbo-odoo** [comment]
@robodoo retry

**Comment by dbo-odoo** [comment]
@robodoo  retry

**Pull request closed without merging**